docker/clustering-script/get_clusters.py

```diff
@@ -6,9 +6,12 @@
 import clickhouse_connect.driver
 import clickhouse_connect.driver.client
 import numpy as np
-from sklearn.cluster import KMeans
+from sklearn.cluster import HDBSCAN
 from scipy.spatial.distance import cosine
-anthropic_client = anthropic.Anthropic()
+import dotenv
+
+dotenv.load_dotenv()
```

Comment:
```suggestion

```
Need to remove dotenv so that the env vars work correctly in docker

Reply:
This is fine